composer.json

```diff
@@ -126,7 +126,7 @@
         "prestashop/statsstock": "^2",
         "prestashop/translationtools-bundle": "^5.0",
         "sensio/framework-extra-bundle": "^5.1",
-        "smarty/smarty": "^4.1.0",
+        "smarty/smarty": "^4.3.0",
```

Comment:
Did you check there is no BC breaks between the two? (I know it's not supposed to but... 😅 )